filament/backend/src/vulkan/VulkanContext.cpp

```diff
@@ -459,13 +462,13 @@ void acquireSwapCommandBuffer(VulkanContext& context) {
     // Restart the command buffer.
     VkCommandBuffer cmdbuffer = swap.commands.cmdbuffer;
     VkResult error = vkResetCommandBuffer(cmdbuffer, 0);
-    ASSERT_POSTCONDITION(not error, "vkResetCommandBuffer error.");
```

Comment:
uh! what was that??! :-)

Reply:
`not` is legal C++ and even conforms to Pixar's code conventions :)